src/packages/client/src/__tests__/benchmarks/huge-schema/huge-schema.bench.ts

```diff
@@ -38,4 +39,24 @@ suite
     // Output benchmark result by converting benchmark result to string
     console.log(String(event.target))
   })
+  .on('complete', () => {
+    getSize('@prisma/client')
+    getSize('.prisma/client')
```

Comment:
So here you check 
- the size of npm package @prisma/client
- the size of folder `.prisma/client` which contains the query engine 
And they are tracked as one metric or 2? (separately or together?)

Reply:
https://prisma.github.io/prisma/dev/bench/